src/apps/inventory/redis_utils.py

```diff
@@ -0,0 +1,14 @@
+import redis
+
+
+class RedisConnection:
+    _instance = None
+
+    def __new__(cls, host="localhost", port=6379, db=0):
```

Comment:
Maybe it is better to store arguments somewhere in variables to have more flexibility

Reply:
For now i'm not sure is it correct way to make Redis connection. The other way can be write CACHE configuration in settings. Once, it will be estimated as a correct path, i'll definitely add arguments to .env

Reply:
Take a look at django recommendations

I can suggest:
# REDIS_CACHE_DB=
# REDIS_CACHE_HOST=
# REDIS_CACHE_PORT=
# REDIS_CACHE_TLS=